### PR TITLE
CLOUDP-187645 Resetting tags is not working on Atlas CLI

### PR DESCRIPTION
**Pull request by fmenezes**
## Proposed changes
Resetting tags is not working on Atlas CLI

_Jira ticket:_ CLOUDP-187645

Closes #[issue number]

## Checklist

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

**Comment by tibulca** [comment]
can you please implement the same change for quickstart too: https://github.com/mongodb/mongodb-atlas-cli/blob/771b67a149787aec6e9fbb4daad1215ef0d15364/internal/cli/atlas/quickstart/cluster_setup.go#L116

**Review by tibulca — COMMENTED** [review]
can you please implement the same change for quickstart too: https://github.com/mongodb/mongodb-atlas-cli/blob/771b67a149787aec6e9fbb4daad1215ef0d15364/internal/cli/atlas/quickstart/cluster_setup.go#L116

File: internal/cli/atlas/serverless/update.go [hunk 1]
```diff
@@ -71,8 +71,13 @@ func (opts *UpdateOpts) newServerlessUpdateRequestParams() *atlasv2.ServerlessIn
...
 	}
 
+	if len(opts.tag) > 0 {
+		params.Tags = []atlasv2.ResourceTag{}
+	}
 	for k, v := range opts.tag {
-		params.Tags = append(params.Tags, atlasv2.ResourceTag{Key: pointer.Get(k), Value: pointer.Get(v)})
+		if k != "" && v != "" {
+			params.Tags = append(params.Tags, atlasv2.ResourceTag{Key: pointer.Get(k), Value: pointer.Get(v)})
+		}
```
Comment (colm-quinn) [inline internal/cli/atlas/serverless/update.go:80]: Does this mean that tags are always appended now (no replace all on the update command?)

If yes, we should also update the docs - I think it's a more predictible behaviour

Comment (fmenezes) [reply internal/cli/atlas/serverless/update.go:80]: it is a replace @colm-quinn we don't fetch current state

**Review by tibulca — APPROVED** [review]

**Pull request merged**